### PR TITLE
[.github meta resource] Refactor current bug template to request more information

### PR DESCRIPTION
**Pull request by fktkrt**
Signed-off-by: fktkrt <fktkrt@gmail.com>

#### What this PR does / why we need it:

This PR extends the bug-temaplate currently in use with useful context informations. The goal of this PR to acquire more details for each bug reports, hence fixing the issue faster.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped (although this might not be needed in this case)
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by torstenwalter — CHANGES_REQUESTED** [review]
Can you revert changes in Chart.yaml. it's not necessary to bump a chart version of you only modify files outside of charts directory.

**Comment by fktkrt** [comment]
> Can you revert changes in Chart.yaml. it's not necessary to bump a chart version of you only modify files outside of charts directory.

Sure, it's done.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -40,31 +40,34 @@ please put the output of it here
...
 
-```
+**The helm command that you execute and failing/misfunctioning**:
+
+For example:
+
+```console
 helm install my-release prometheus-community/name-of-chart --version version --values values.yaml
-```
+``**
```
Comment (torstenwalter) [inline .github/ISSUE_TEMPLATE/bug_report.md:65]:
```suggestion
```
```

To close the console block from above.

**Comment by gkarthiks** [comment]
seems @fktkrt you have to do the DCO

**Review by gkarthiks — COMMENTED** [review]
/lgtm, just the DCO is blocking

**Review by monotek — APPROVED** [review]

**Review by torstenwalter — CHANGES_REQUESTED** [review]
@fktkrt you marked https://github.com/prometheus-community/helm-charts/pull/326/files#r519758839 as resolved, but I think the issue is still present.

**Comment by fktkrt** [comment]
> @fktkrt you marked https://github.com/prometheus-community/helm-charts/pull/326/files#r519758839 as resolved, but I think the issue is still present.

You're right, sorry, it's fixed now.
I hope the effects of these changes are are making up the review time for this simple PR in the long run. :)

**Review by torstenwalter — APPROVED** [review]

**Comment by torstenwalter** [comment]
@fktkrt If you would know how often I missed something ;-)

Thanks for the improved template 💯

**Pull request merged**